bot/converters.py

```diff
@@ -574,6 +574,55 @@ async def convert(self, ctx: Context, arg: str) -> t.Optional[dict]:
                 raise e
 
 
+class DayDuration(Converter):
+    """
+    Convert a string representing day time (hours and minutes) to a UTC datetime object.
+
+    The hours and minutes would be combined with UTC day if no 'am' or 'pm' is passed with
+    the string, then it is assumed that the time is in 24 hour format.
+
+    The following formats are accepted:
+        - H:M
+        - H:Mam/pm
+        - HMam/pm
+        - Ham/pm
+        - H
+
+    where `H` represents Hours and `M` represents Minutes.
+    """
+
+    TIME_RE = re.compile(
+        r"^(1[0-2]|0?[1-9]):?([0-5][0-9])? ?([AaPp][Mm])$"  # Twelve hour format
+        "|"
+        r"^([0-9]|0[0-9]|1[0-9]|2[0-4]):?([0-5][0-9])?$"   # Twenty four hour format
+    )
+
+    async def convert(self, _ctx: Context, argument: str) -> datetime:
```

Comment:
Is there a reason this is `_ctx` instead of simply `ctx` as in our other converters?

Reply:
Nope, I just copied one of the earlier convertors as a template and applied the changes.

Reply:
Should probably be changed to `ctx` then

Reply:
Oh, my bad, didn't look at the source when was commenting on this, it is `_ctx` since I am not using it anywhere.